README.md

```diff
@@ -25,7 +25,7 @@ DesignGuide requires [Node.js](https://nodejs.org/en/) (includes npm).
 ### Installing
 
 1. Open a terminal and navigate to repo root
-2. `npm install`
+2. `npm ci`
```

Comment:
I don't think this should be used for actual development
```suggestion
2. `npm i`
```

Reply:
We don't want all developers to touch packages or package-lock, thereby `npm ci`

Only developers actively looking to update these files will run `npm install`

Reply:
yes